src/Controllers/EventListenerController.ts

```diff
@@ -67,4 +68,8 @@ export class EventListenerController {
     }
     this.dispatcher.dispatchEvent({ eventType: EventType.POINTER_MOVE, e }, EventOrigin.USER);
   };
+
+  private onStrokePropertiesChange = (strokeProps: IStrokePropOptions) => {
```

Comment:
This should not be in EventListenerController. A similar function can be found in Service.ts already.